AggregateMetrics/AggregateMetrics/AzureWebApp/AzureWebApp.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.Two;
+using System;
+using Microsoft.ApplicationInsights.DataContracts;
+using System.Net.Http;
+using System.Runtime.Caching; 
+
+namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+
+    class PerformanceCounterGauge : ICounterValue
+
+    {
+        public string GetJson()
+        {
+            HttpClient client = new HttpClient();
+            var t = client.GetStringAsync("http://remoteenvironmentvariables.azurewebsites.net/api/EnvironmentVariables/WEBSITE_COUNTERS_APP/");
+            t.Wait();
+
+            return t.Result;
+        }
+
+        public string GetValue(string json)
+        {
+            string prefix = "\\\"privateBytes\\\":";
```

Comment:
please create a separate class that will have a method `GetRawCounterValue(string name)' that will incapsulate cache and will not have `privateBytes` hardcoded